### PR TITLE
B3P classification model

### PR DESCRIPTION
**Pull request by maykcaldas**
Uploading the training script, the checkpoint, and the log with the metrics.

### CODE REVIEW

File: experiments/B3P_clas.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+        test_metrics = test(model, criterion, test_loader)
+        print(f"\tTest Loss: {test_metrics['loss']}")
+        print(f"\tTest Accuracy: {test_metrics['accuracy']}")
+        print(f"\tTest ROC AUC: {test_metrics['roc_auc']}")
+
+        # torch.save(model.state_dict(), "B3P_classifier.pt")
+
+if __name__ == "__main__":
+    torch.manual_seed(42)
```
Comment (GiaJordan) [inline experiments/B3P_clas.py:118]: question, do we want to still set a seed now that we've moved out of debugging?

**Review by GiaJordan — APPROVED** [review]
